app/models/feed/apple_subscription.rb

```diff
@@ -0,0 +1,90 @@
+class Feed::AppleSubscription < Feed
+  DEFAULT_FEED_SLUG = "apple-delegated-delivery-subscriptions"
+  DEFAULT_TITLE = "Apple Delegated Delivery Subscriptions"
+  DEFAULT_AUDIO_FORMAT = {"f" => "flac", "b" => 16, "c" => 2, "s" => 44100}.freeze
+
+  after_initialize :set_defaults
+
+  has_one :apple_config, class_name: "::Apple::Config", dependent: :destroy, autosave: true, validate: true
+  has_one :key,
+    through: :apple_config,
+    class_name: "Apple::Key",
+    dependent: :destroy,
+    foreign_key: :key_id
+
+  accepts_nested_attributes_for :apple_config, allow_destroy: true, reject_if: :all_blank
+  accepts_nested_attributes_for :key, allow_destroy: true, reject_if: :all_blank
+
+  validate :unchanged_defaults
+  validate :only_apple_feed
+  validate :must_be_private
+
+  def set_defaults
+    self.slug ||= DEFAULT_FEED_SLUG
+    self.title ||= DEFAULT_TITLE
+    self.audio_format ||= DEFAULT_AUDIO_FORMAT
+  end
```

Comment:
You should call `super` here, since the parent `Feed.set_defaults` has some things this also needs (file name, etc).